reporters/reporter-default/src/main/java/org/radargun/reporting/html/TimelineChart.java

```diff
@@ -97,7 +97,7 @@ public void setEvents(List<? extends Timeline.Event> events, int slaveIndex, lon
       for (Timeline.Event event : events) {
          if (event instanceof Timeline.Value) {
             Timeline.Value value = (Timeline.Value) event;
-            int bucket = (int) ((event.timestamp - startTimestamp) * MAX_EVENT_VALUES / (endTimestamp - startTimestamp));
+            int bucket = (int) ((event.timestamp - startTimestamp) * (MAX_EVENT_VALUES-1) / (endTimestamp - startTimestamp));
```

Comment:
Not exactly, I was not looking into this too deeply, but from what I did look at, when you have a very specific timestamp combination the result, which is used as index to store something to, will equal  MAX_EVENT_VALUES, which is also used for a size of array the index is used on, resulting in ArrayIndexOutOfBounds exception. It happens only occasionally, I believe it was an oversight.